### DISCUSSION TITLE
Possible bug in the INDI LX200 INDI Interface

### DISCUSSION THREAD

**Discussion by wbirk** [Stellarium/stellarium#3024] [General]
hi!
i wrote this controller - https://lacerta-optics.com/BlueLizard#m - which interfaces via INDI to stellarium and other programs. it appears to me that the LX200 - Basic driver interface does not issue a :Q# command when a slew is to be aborted. in fact, i do not get anything when "Abort slew" is pressed ... 
yours wolfi (wbirkfellner@gmail.com)

**Comment by 10110111** [comment]
This command doesn't seem to be implemented in `TelescopeClientINDI`. I suppose this should be reported as an issue if the protocol is supposed to support it.

**Reply by wbirk** [reply]
hi! well - it is the method of stopping a slew in LX 200 and there is a button to do that - what is the best way to report it?
yours wolfi

**Reply by 10110111** [reply]
Open a new issue [here](https://github.com/Stellarium/stellarium/issues) in the issues section.

**Reply by gzotti** [reply]
For the lifetime of this project, the telescope control protocol has never implemented all possible commands. 

Some while ago we had a request to implement AbortSlew. PR #2754 is the place to do that. I can observe that the Telescope Simulator aborts its slew when so commanded.  I have a NEQ3 with Lacerta MGEN, but don't find time currently to use it, therefore I did not continue on that plugin. Whoever owns a goto telescope, a computer, and has a little idea of programming may be motivated enough to take up this task. Especially as I did not continue in INDI or ASCOM. We could really need a dedicated developer who can invest a few rainy weekends (and half-usable nights) to fix and test a series of issues currently existing in the TC plugin.
